### ISSUE TITLE
Deathchill Servant incorrect behavior and damage values.

### ISSUE THREAD

**Issue by balakethelock** [slowtorta/turtlewow-bug-tracker#3420]
**Please, provide a clear description what the bug is:**
Deathchill Servants are mobs that "Deathknight" mobs in Naxxramas can summon using the Raise Dead spell.
In turtle wow, the autoattack damage of Deathchill Servants is hugely overtuned, and they spawn with zero threat.

**Steps to reproduce the behavior with as much detail as possible:**

1. Engage combat with the first pack of deathknight wing (three mobs, one of which is a Deathknight)
2. Kill one of the smaller adds to make the deathknight use his Raise Dead ability
3. See that the Deathchill Servants start at 0 threat and have their own threat table.
4. Also see that the Deathchill Servants autoattack damage is hugely overtuned. Double that of most other npcs in naxxramas. The database puts it at 3,670 - 3,740, meanwhile the Deathknights who summoned them deal 2,745 - 2,815

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=16146
NPC: https://database.turtle-wow.org/?npc=16390
Spell: https://database.turtle-wow.org/?spell=28353

**Expected behavior. Describe how it should work:**

1. Deathchill Servants are guardians. Meaning that they are supposed to spawn with the threat table of the deathknight who summoned them, and go on the person tanking the Deathknight unless taunted or overthreated directly.

2.Their autoattack damage should be reduced to bring them closer to blizzlike values. Here is some old Warcraft Logs data where the autoattack damage of Deathchill Servants is logged as "Raise Dead" damage

https://i.imgur.com/wEmNNXd.png a bear gets hit for 860 on 70% armor
https://i.imgur.com/YzEbwll.png a rogue gets hit for 1.6k on 33% armor
From this we can deduce that the true swing damage should be around 2400 - 2800

In turtle wow, the mob does 3670 - 3740, here is a more detailed view of it in practice, damage dealt by the mob https://i.imgur.com/Rh9nkyj.png
If you want to check out the logs yourself, https://legacyplayers.info/viewer/2930/detail/2/0

TLDR: the mob is starting the fight with threat on the tank, and is doing around 30% more damage than it should.
